php/libraries/NDB_Page.class.inc

```diff
@@ -601,6 +601,7 @@ class NDB_Page
                   $baseurl . "/js/util/queryString.js",
                   $baseurl . '/js/components/Form.js',
                   $baseurl . '/js/components/Markdown.js',
+                  $baseurl . '/js/components/Tabs.js',
```

Comment:
There is a similar line like this in the instrument_builder php file. That should be removed as well as its not needed when imported